yarn.lock

```diff
@@ -6636,6 +6636,14 @@ ember-cli-get-component-path-option@^1.0.0:
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/ember-cli-get-component-path-option/-/ember-cli-get-component-path-option-1.0.0.tgz#0d7b595559e2f9050abed804f1d8eff1b08bc771"
 
+ember-cli-github-pages@^0.2.2:
```

Comment:
not needed.

Reply:
Hi... Can I delete this file?

Reply:
wait...what? are u saying to remove yarn file? 
>Hi... Can I remove this file?

NO

Reply:
> wait...what? are u saying to remove yarn file?

Oh sorry, my mistake. I am kind of naive here.